test/test_tree_2d_euleracoustics.jl

```diff
@@ -0,0 +1,21 @@
+module TestExamples2DEulerAcoustics
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2d_dgsem")
```

Comment:
```suggestion
EXAMPLES_DIR = joinpath(get_examples(), "tree_2d_dgsem")
```

Reply:
Shouldn't this be
```suggestion
EXAMPLES_DIR = joinpath(examples_dir(), "tree_2d_dgsem")
```
?